src/pages/docs/guides/chat/build-embedded-chat.mdx

```diff
@@ -0,0 +1,414 @@
+
+Components handle message display with history loading, editing and deletion, reactions, typing indicators, and presence. [Providers](/docs/chat/react-ui-kit/providers) manage themes, avatars, and chat settings. See the [getting started guide](/docs/chat/getting-started/react-ui-kit) for setup details.
+
+## Enriching experiences with Ably's realtime services
+
+Ably's comprehensive platform enables you to combine chat with other realtime features to create rich, interactive experiences.
+
+[**Pub/Sub channels**](/docs/basics) enable you to build rich, interactive experiences:
+- **Live polls:** Quick feedback during collaborative decisions.
+- **Status updates:** Real-time progress indicators for tasks.
+- **Interactive ratings:** Instant feedback collection.
+- **Live auctions:** Real-time bidding in marketplace scenarios.
+
+These combined services transform basic chat into comprehensive task management platforms, where communication, coordination, and real-time updates work together seamlessly.
+
+## Moderation: maintaining quality conversations
+
+Effective moderation ensures your chats remain professional, safe, and productive. Maintaining conversation quality is crucial for providing a pleasant online environment and keeping your users safe.
+
+Ably supports [moderating messages](/docs/chat/moderation) both before and after publication, making it easy to integrate with AI-powered or human moderation systems.
+
+* **After-publish moderation:** Messages appear instantly, then are removed if flagged as inappropriate. Best for scenarios where immediacy matters. It can be used to detect abusive behavior or inappropriate content after the fact.
+* **Before-publish moderation:** Messages are held until approved. Use this for high-stakes scenarios where every message must be vetted.
+
+### Key moderation considerations for chat
+
+1. **Platform standards**
+   * What level of moderation is appropriate for your audience?
+   * How will you handle different types of content? For example:
+     * **Hate speech and harassment:** Detecting discriminatory language, threats, or targeted abuse.
+     * **Inappropriate content:** Flagging adult content, violence, or graphic material.
+     * **Toxicity:** Measuring overall message sentiment and hostility.
+
+2. **Technical integration**
+   * **Latency impact:** AI moderation adds up to 100ms to message delivery.
+   * **Integration options:** Choose from pre-built integrations or connect your existing moderation systems via webhooks, serverless functions, or queues.
+
+3. **Context-specific approaches**
+   * **Customer support:** Protect both customers and agents from abuse and harassment.
+   * **Gaming:** Prevent harassment while allowing enthusiastic expressions.
+   * **Marketplace:** Ensure rules are being followed such as detecting if outside-platform contact info is exchanged.
```

Comment:
This is pushing more cognitive load onto the customer by listing questions rather than solutions.

Reply:
It's the same style as the livestream moderation section. Should we change both in some way?